server.js

```diff
@@ -16,15 +16,15 @@ client.on('error', err => console.error(err));
 app.use(express.static('./public'));
 
 app.get('/', function(request, response) {
-  response.sendFile('./public/index.html', {root: './public'});
+  response.sendFile('./index.html', {root: './public'});
```

Comment:
FYI you do not need `./` in front of the file you are sending since the directory is defined in `root`.